opengcs/README.md

```diff
@@ -1,38 +0,0 @@
-

```

Comment:
Are we putting this content in the main readme?

Reply:
Yea, was curious how we wanna word things now, as the readme already seems pretty bare given how much hcsshim actually contains. Might be a good chance to revamp the README in general

Reply:
A good start might be just putting the opengcs README in as a top-level section.